### DISCUSSION TITLE
General winget error

### DISCUSSION THREAD

**Discussion by dlundahl91** [Romanitho/Winget-Install#89] [Q&A] [answered]
Hey team

Has any of you seen this problem before?
Please see attached image.
Any ideas?

<img width="486" alt="Screenshot 2023-10-07 191940" src="https://github.com/Romanitho/Winget-Install/assets/144660401/d7570b05-01c9-4ed8-972d-c3206e38901c">

**Comment by KnifMelti** [comment]
This is general and probably global MS source problem in this moment too.
Has happened before...

**Reply by KnifMelti** [reply, marked as answer]
Got it with winget v1.5.2201, is resolved with v1.6.2771.
Update your winget
